ecaudit/src/main/java/com/ericsson/bss/cassandra/ecaudit/auth/PermissionChecker.java

```diff
@@ -80,11 +80,7 @@ void checkAlterRoleAccess(AuthenticatedUser performer, RoleResource role, RoleOp
 
     private boolean isChangingPasswordOfOtherRole(AuthenticatedUser performer, RoleResource role, RoleOptions options)
     {
-        if (options.getPassword().isPresent())
-        {
-            return !performer.getName().equals(role.getRoleName());
-        }
-        return false;
+        return options.getPassword().isPresent() && !performer.getName().equals(role.getRoleName());
```

Comment:
I'm not saying the previous way was looking good, or were easy to read. But IMO the one-liner is even harder to read, so I'm -1 on this.